src/main/java/com/example/demo/security/authInfo/AuthRequest.java

```diff
@@ -2,16 +2,17 @@
 
 import javax.validation.constraints.NotNull;
 
+import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.Setter;
 import org.hibernate.validator.constraints.Length;
 
-@Getter @Setter
+@Getter @Setter @AllArgsConstructor
 public class AuthRequest { // POJO
     @NotNull @Length(min = 1, max = 50)
     private String username;
 
-    @NotNull @Length(min = 1, max = 10)
+    @NotNull @Length(min = 1, max = 50)
```

Comment:
Good idea changing the max here 👍 Don't think I have many passwords under 10 characters 😆

Reply:
I'm going to make better validations for those later methinks